src/config/content/mycomp_data/team.js

```diff
@@ -0,0 +1,74 @@
+export const TeamMembers = [
```

Comment:
instead of writing in `mycomp_data`, create a dir `content/teamsData/`

Reply:
Added to the specified path